website/content/en/docs/getting-started/_index.md

```diff
@@ -52,12 +52,28 @@ AWS_ACCOUNT_ID=$(aws sts get-caller-identity --query Account --output text)
 
 ### Create a Cluster
 
-Create a cluster with `eksctl`. The [example configuration](eksctl.yaml) file specifies a basic cluster (name, region), and an IAM role for Karpenter to use.
+Create a cluster with `eksctl`. This example configuration file specifies a basic cluster with one initial node and sets up an IAM OIDC provider for the cluster to enable IAM roles for pods:
 
 ```bash
-curl -fsSL https://karpenter.sh/docs/getting-started/eksctl.yaml \
-  | envsubst \
-  | eksctl create cluster -f -
+cat <<EOF > cluster.yaml
```

Comment:
Iirc it's possible to pass the file directly to eksctl inline to avoid dropping a file on the local filesystem.

Reply:
It is and would work here. Personally I just always want to take a look at the config file before launching, because it takes like 20min to launch cluster + ng and some things are annoying to change later. I wouldn’t object to going back to the `eksctl -f -` option though.

Reply:
For consistency with the rest of the guide, I'd probably lean doing it all inline. 

> Personally I just always want to take a look at the config file before launching

I often edit them inline after pasting but before entering them.

> Also, sometimes multi-line pasting can subtly break depending on the terminal emulator, etc.

This is a decent point, but at some point, there's only so much we can do to make this smooth with all user environments